README.md

```diff
@@ -178,11 +178,11 @@ access to the management interface.
 
 ### Configure openvpn-monitor
 
-Copy the example configuration file `openvpn-monitor.conf.example` to the same
+Copy the example configuration file `openvpn-monitor.conf` to the same
 directory as openvpn-monitor.py.
```

Comment:
This was added due to a request in https://github.com/furlongm/openvpn-monitor/issues/33

When running from source in a git repo, it makes sense, as a git pull fails if the conf has been modified.

Reply:
But you have to create a copy of the config when running from source in a Git repo. #33 looks like insufficient scripting knowledge to me. It's really not that hard to script "git pull" in a way that handles conflicts.

Anyway, it's your decision. Merge or reject as you please. It's only 1/4 commits anyway.

Reply:
Also, git pull need not always fail if the conf file has been modified. It might also conveniently merge in changes.